platform/source/mbed_retarget.cpp

```diff
@@ -889,11 +1061,15 @@ extern "C" int fsync(int fildes)
     } else {
         return 0;
     }
+#else
+    return -1;
+#endif // !(MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY)
 }
 
 #ifdef __ARMCC_VERSION
 extern "C" long PREFIX(_flen)(FILEHANDLE fh)
 {
+#if !(MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY)
```

Comment:
Same question: can you remove _flen entirely?

Reply:
The function is not `_flen` but `_sys_flen`. If any of the system I/O functions are redefined, then they must all be redefined. So I cannot remove it.